python/lsst/sims/maf/batches/metadataBatch.py

```diff
@@ -6,7 +6,7 @@
 import lsst.sims.maf.plots as plots
 import lsst.sims.maf.metricBundles as mb
 from .colMapDict import ColMapDict
-from .common import standardSummary, extendedMetrics
+from .common import standardSummary, extendedMetrics, filterList
 
 __all__ = ['metadataBasics', 'allMetadata']
 
```

Comment:
Should `filterlist` be removed as a `kwarg` from `metadataBasics` now that it is imported from common?

Reply:
That seems like a good idea.